build.gradle

```diff
@@ -421,6 +270,8 @@ task nightlyFunctional(type: Test, description: 'Runs the functional tests.', gr
 
             systemProperty 'AUTH_PROVIDER_SERVICE_CLIENT_KEY', getVaultSecret('cms-service-key')
             systemProperty 'AUTH2_CLIENT_SECRET', getVaultSecret('idam-secret')
+            // systemProperty 'IDAM_API_BASEURL', 'http://localhost:5000'
+            // systemProperty 'CASE_DATA_STORE_BASEURL', 'http://localhost:4452'
```

Comment:
Do these commented lines need to stay?

Reply:
For now, I will do a separate PR to change the way we toggle between testing locally vs testing in AAT